### ISSUE TITLE
screenreader improvements 

### ISSUE THREAD

**Issue by aryanchoudharypro** [Genymobile/scrcpy#4808]
- [ ] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ ] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 
 - scrcpy version: 2.4
 - installation method: Windows release
 - device model: realme gt master adition 
 - Android version: 14 evolution x cr

so, when useing this tool, i am not able to send the keys wich re used for taalkback 
another thing is add something through wich we as users of screenreaders on windows can copy files to the scrcpy windows as we can not drop them.

**Comment by rom1v** [comment]
> we as users of screenreaders on windows can copy files to the scrcpy windows as we can not drop them.

If you drop a file from your computer to the scrcpy window, it should send it to the device;

https://github.com/Genymobile/scrcpy/blob/master/doc/control.md#push-file-to-device

**Comment by aryanchoudharypro** [comment]
How can I drop the files like can I use the control c and control v to do
this as I am blind and hence use a screen reader and cannot drag or drop
the files

**Comment by rom1v** [comment]
Sorry, you can't send a file with Ctrl+c/Ctrl+v (which is limited to clipboard text synchronization).

**Comment by aryanchoudharypro** [comment]
What about the TalkBack issue

**Comment by aryanchoudharypro** [comment]
Is there a way to disable all the shortcuts

**Comment by rom1v** [comment]
> What about the TalkBack issue

See #1615. Maybe also try UHID keyboard: https://github.com/Genymobile/scrcpy/blob/master/doc/keyboard.md#physical-keyboard-simulation

> Is there a way to disable all the shortcuts

No.

**Comment by aryanchoudharypro** [comment]
Then I use OTG mode it works great with TalkBack

**Comment by aryanchoudharypro** [comment]
Okay so doing these things work for me. First you should have u h i d
keyboard selected and then you should also select the mod key to which ever
you don't use for example I selected the right control key and I don't use
it offen and then it will work
